query-api/src/main/java/com/brandwatch/minibcr/queryapi/config/RedisConfig.java

```diff
@@ -0,0 +1,23 @@
+package com.brandwatch.minibcr.queryapi.config;
+
+import java.time.Duration;
+
+import org.springframework.cache.annotation.EnableCaching;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.redis.cache.RedisCacheConfiguration;
+import org.springframework.data.redis.serializer.GenericJackson2JsonRedisSerializer;
+import org.springframework.data.redis.serializer.RedisSerializationContext;
+
+@Configuration
+@EnableCaching
+public class RedisConfig {
+
+    @Bean
+    public RedisCacheConfiguration cacheConfiguration() {
```

Comment:
I am confused about this Redis dependency you're introducing here. As far as I can see you don't use it anywhere.

Reply:
Apologies, I just noted the `@Cacheable` annotation in the repo. It's clearer now 👍